frontend/src/components/dialog/DialogActivityEdit.vue

```diff
@@ -42,10 +42,11 @@ export default {
       ],
       embeddedEntities: [
         'category'
-      ],
+      ]
+      /*
       embeddedCollections: [
         'scheduleEntries'
-      ]
+      ] */
```

Comment:
Wieso das? Wieder aktivieren oder ganz löschen.

Reply:
To be fixed in #2350